src/language-js/print/statement.js

```diff
@@ -168,14 +168,19 @@ const isClassProperty = ({ type }) =>
   type === "ClassProperty" ||
   type === "PropertyDefinition" ||
   type === "ClassPrivateProperty" ||
-  type === "ClassAccessorProperty";
+  type === "ClassAccessorProperty" ||
+  type === "AccessorProperty" ||
+  type === "TSAbstractAccessorProperty";
```

Comment:
Can you investigate why `TSAbstractPropertyDefinition` not added here, or why `TSAbstractAccessorProperty` should be added here?

Reply:
I guess it's just missed.

Reply:
If I add `"TSAbstractPropertyDefinition"` to this list, the test does not fail. So it is maybe missed.

Reply:
What a `TSAbstractPropertyDefinition` look like


```js
class A {
	abstract get;
	foo() {}
}
```

?

We can test it.

Reply:
Yes, missed [playground](https://prettier.io/playground/#N4Igxg9gdgLgprEAucAbAhgZ0wAgII7AA6URM6ARpjAE7pgw4DmcMA3CWQGYQQAUASkIBfEsJAAaEBAAOMAJbRMyUOho0IAdwAKahMpTpUm9AE9lUinTABrVgGV0AWzgAZeVDjIuRzHEvWdjD2MvQeTMi0AK7+IHBOFHAAJknJruhQTFHoLABiEDRO6DAKmcgg6FEwEJIgABYwTqgA6nXy8JihYHD2+u3yAG7tpuVg2LUefjQw2nRMRd6+sQBWmAAe9uGocACKURDwi6h+UqE0U+UwpjJwmGA08nK1Mg+wzfJJMHXIABwADKcNH5mnQZOUXrc4DQBl4pABHfbwWayAwVTAAWk8yWStRocAR8jxsxyCyQPmOsT8TnkRxOIEwWzgeBKDwoVTg2ih7k8tMpjL2By8ZKWUnIFHen2+SAATKK6PJUOEAMIQJykuKYACstSifgAKpQDOS6QMYgBJKCpWD2e6PGB4S32K7bXnCYRAA)